rustls/Cargo.toml

```diff
@@ -34,6 +34,7 @@ logging = ["log"]
 aws_lc_rs = ["dep:aws-lc-rs", "webpki/aws_lc_rs"]
 aws-lc-rs = ["aws_lc_rs"] # Alias because Cargo features commonly use `-`
 ring = ["dep:ring", "webpki/ring"]
+web = ["ring/wasm32_unknown_unknown_js"]
```

Comment:
Isn't this something that could be activated outside of rustls by a dependent crate that wanted to use *ring* by taking its own direct dependency?

If a feature for this were to be added here, should it also activate `pki-types/web`?

Reply:
Also should this be `ring?/wasm32_unknown_unknown_js` to avoid enabling the ring dependency in scenarios where the aws-lc-rs provider is actually selected?

Reply:
> Isn't this something that could be activated outside of rustls by a dependent crate that wanted to use `*ring*` by taking its own direct dependency w/ the feature enabled?
> 
> If a feature for this were to be added here, should it also activate `pki-types/web`?

Yes indeed, they don't have the feature yet but I made a PR, let's wait to see if they merge it

Reply:
> Also should this be `ring?/wasm32_unknown_unknown_js` to avoid enabling the ring dependency in scenarios where the aws-lc-rs provider is actually selected?

I thought it would only enable ring if I wrote it that way `web = ["dep:ring", "ring/wasm32_unknown_unknown_js"]` but I will check